src/tools/rbd/action/Snap.cc

```diff
@@ -255,13 +260,13 @@ int execute_remove(const po::variables_map &vm) {
   librados::Rados rados;
   librados::IoCtx io_ctx;
   librbd::Image image;
-  r = utils::init_and_open_image(pool_name, image_name, "", false, &rados,
+  r = utils::init_and_open_image(pool_name, image_name, snap_name, false, &rados,
```

Comment:
The all operations in tools/rbd/Snap.cc are working on snapshot. So I think we can get a context for a specified snapshot. In this case, I need to get imctx.snap_id later.
```
parent_spec parent_spec(ictx->md_ctx.get_id(), ictx->id, ictx->snap_id);
```
https://github.com/ceph/ceph/pull/10087/files#diff-254555dde8dcfb7fb908791ab8214b92R874